testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/ssl/AutomaticSelfSignedCertificateGenerationTestCase.java

```diff
@@ -96,7 +96,7 @@ public static WebArchive createDeployment() {
 
     @Test
     public void testSelfSignedCertificateGenerated() throws Exception {
-        final URL servletUrl = new URL("https", TestSuiteEnvironment.getServerAddress(), HTTPS_PORT,
+        final URL servletUrl = new URL("https", TestSuiteEnvironment.getServerAddress(), TestSuiteEnvironment.getHttpPort(),
```

Comment:
@darranl This is the reason why this test started failing in the CI. This seems to be a typo where instead of HTTPS port we are now communicating with HTTP port. How should I fix this here? Should I checkout Farah's branch and rewrite this or create separate PR?

Reply:
@Skyllarr Thank you for tracking this down, for this issue I would say checkout Farah's branch and add a commit of your own with the fix and then submit a PR containing everything so we can get it all onto CI.

Reply:
@darranl I see, thanks I'll do that.